web/src/ui/timeline/content/PolicyRuleBody.tsx

```diff
@@ -0,0 +1,28 @@
+import { PolicyRuleContent } from "@/api/types"
```

Comment:
Don't forget to add the copyright headers

Reply:
Copyright headers are missing from index.ts too, should I add them there too?